xarray/core/formatting_html.py

```diff
@@ -23,7 +23,7 @@
 def _load_static_files():
     """Lazily load the resource files into memory the first time they are needed"""
     return [
-        read_binary(package, resource).decode("utf-8")
+        files(package).joinpath(resource).read_bytes().decode("utf-8")
```

Comment:
```suggestion
        files(package).joinpath(resource).read_text(encoding="utf-8")
```
any reason why we read bytes and then decode it?

Reply:
good catch! I'm not sure, I just copied the `read_binary().decode(...)` pattern from before.

Reply:
I think the pattern has been carried over from when we used `pkg_resources.resource_string`, because that returns bytes. No need to keep that so let's go with your suggestion.